post-types/templates/dashboard/single-dashboard.php

```diff
@@ -19,7 +19,7 @@
 	<div class="container">
 		<div class="row">
 			<div class="four columns" style="margin-left: 0px;">
-				<h1 id="region_name">Myanmar</h1>			
+				<h1 id="region_name" class="region_name_container">Myanmar</h1>			
```

Comment:
@DragonVirus What is the difference between this template and **single-dashboard-my-overview.php**? shouldn't this one be ocmpletely generic?